doc/config.toml.tmpl

```diff
@@ -25,6 +25,14 @@ pygmentsUseClasses = true
   github_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
   github_repository_edit = "https://github.com/TheThingsNetwork/lorawan-stack/edit/master/doc/content"
   version = "{{.CurrentVersion}}"
+  [params.intercom]
+    enabled = false
+    appid = ""
+  [params.feedback]
+    enabled = false
+  [params.analytics]
+    enabled = false
+    trackingid = ""
```

Comment:
Is this really a Trackingid? Or should it be `tracking_id`?

Reply:
Also, isn't this already built in to Hugo?

Reply:
google analytics is built in but we need a different script to track events